scripts/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+docker stop icvtt
+docker rm icvtt
+docker run -d -p 8080:8080 -v /opt/pyvtt/prod:/opt/pyvtt/prod --restart=on-failure --name icvtt pyvtt:latest
```

Comment:
do we want to add the `--no-logs` and `--no-resource-routing` here?

Reply:
Added `--no-logs` to the Dockerfile.